README.md

```diff
@@ -1,10 +1,80 @@
 # terraform-aws-mcaf-landing-zone
-Terraform module to setup and manage various components of the AWS Landing Zone.
+Terraform module to setup and manage various components of the SBP AWS Landing Zone.
 
 Overview of Landing Zone tools & services: 
 
 <img src="images/MCAF_landing_zone_tools_and_services_v040.png" width="600"> 
 
+The SBP AWS Landing Zone consists of 3 repositories:
+- [MCAF Landing Zone module (current repository)](https://github.com/schubergphilis/terraform-aws-mcaf-landing-zone): the foundation of the Landing Zone and manages the 3 core accounts: audit, logging, master
+- [MCAF Account Vending Machine (AVM) module](https://github.com/schubergphilis/terraform-aws-mcaf-avm): providing an AWS AVM. This module sets up an AWS account with one or more Terraform Cloud/Enterprise (TFE) workspace(s) backed by a VCS project
+- [MCAF Account Baseline module](https://github.com/schubergphilis/terraform-aws-mcaf-account-baseline): optional module providing baseline configuration for AWS accounts
+
+# Basic configuration
+```hcl
+module "landing_zone" {
+  source             = "github.com/schubergphilis/terraform-aws-mcaf-landing-zone?ref=VERSION"
+  tags               = var.tags
+
+  control_tower_account_ids = {
+    audit   = "012345678902"
+    logging = "012345678903"
+  }
+}
+```
+
+<!--- BEGIN_TF_DOCS --->
```

Comment:
Should we move the TF docs to the bottom of the readme file? I feel like readability of the documentation gets compromised with the TF docs in here...what do you think?

Reply:
called, and agreed to move down